File: Easy/0001-0099/0001/c/Solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int* result = (int*)malloc(2*sizeof(int));
    for(int i = 0; i < numsSize; i++ ){
        for(int j = i+1; j < numsSize; j++){
            if(nums[i] + nums[j] == target)
            {
                result[0] = i;
                result[1] = j;
                return result;
            }
        }
    }
    result[0] = 0;
	result[1] = 0;
	return result;
}

File: Easy/0001-0099/0001/go/Solution.go
package main

import (
	"fmt"
)

func twoSum(nums []int, target int) []int {
	for i := 0; i < len(nums) - 1; i++ {
		for j := i + 1; j < len(nums); j++ {
			if nums[i] + nums[j] == target {
				return []int{i, j}
			}
		}
	}

	return nil
}

func main() {
	nums := []int{2, 7, 11, 15}
	target := 9

	var res []int
	res = twoSum(nums, target)
	fmt.Println(res)
}